### PR TITLE
Adding documentation for LNS Server Certificate management

### PR DESCRIPTION
**Pull request by danigian**
# PR for issue #775 

## What is being addressed

Adding documentation for LNS Server Certificate management

### CODE REVIEW

**Comment by roel4ez** [comment]
Please add new pages to the TOC by editing the `mkdocs.yml` file. Unfortunately, it does not yet automatically pick up new files (see #796)

When added to the `nav` section, by default it will use the first H1 as page title. 

To specify a different title, use 
```yml
  - 'my-custom-title': user-guide/station-credential-management.md
```
Alternatively, you can specify a different title by adding the following to the top of your new page:
```
---
title: Home
---
```

File: docs/user-guide/station-credential-management.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+As it's possible to read in [official LoRa Basics™ Station documentation](https://doc.sm.tc/station/credentials.html) a Basics™ Station client needs some credentials to establish a secure connection to LNS compatible endpoints.
+
+## Server Authentication
+
+### Importing certificate for server authentication in LoRaWan Network Server module
+
+LoRaWan Network Server IoT Edge module allows to import, from disk, a certificate for server authentication in 'pkcs12' format (.pfx).
+
+Two environment variables need to be set for making this happen:
```
Comment (Mandur) [inline docs/user-guide/station-credential-management.md:13]: Didn't we set this to be optional and we would load at a default location?

Comment (danigian) [reply docs/user-guide/station-credential-management.md:13]: No, optional load is only for Basics Station client module as of now

File: docs/user-guide/station-credential-management.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Basics Station credentials management
+
+Starting with 'Azure IoT Edge LoRaWAN Starter Kit' v2.0.0, the LoRaWan Network Server runs a WebSocket endpoint compatible with [LNS Protocol](https://doc.sm.tc/station/tcproto.html) from LoRa Basics™ Station.
```
Comment (p-schuler) [inline docs/user-guide/station-credential-management.md:3]:
```suggestion
Starting with 'Azure IoT Edge LoRaWAN Starter Kit' v2.0.0, the LoRaWan Network Server runs a WebSocket endpoint compatible with [The LNS Protocol](https://doc.sm.tc/station/tcproto.html) from LoRa Basics™ Station.
```
matching the title of the target link

File: docs/user-guide/station-credential-management.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+As it's possible to read in [official LoRa Basics™ Station documentation](https://doc.sm.tc/station/credentials.html) a Basics™ Station client needs some credentials to establish a secure connection to LNS compatible endpoints.
```
Comment (p-schuler) [inline docs/user-guide/station-credential-management.md:5]:
```suggestion
As described in the official LoRa Basics™ Station documentation - [Credentials](https://doc.sm.tc/station/credentials.html) a Basics™ Station client needs some credentials to establish a secure connection to LNS compatible endpoints.
```

File: docs/user-guide/station-credential-management.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Server Authentication
+
+### Importing certificate for server authentication in LoRaWan Network Server module
+
+LoRaWan Network Server IoT Edge module allows to import, from disk, a certificate for server authentication in 'pkcs12' format (.pfx).
```
Comment (p-schuler) [inline docs/user-guide/station-credential-management.md:11]:
```suggestion
LoRaWan Network Server IoT Edge module allows to import, a certificate in 'pkcs12' format (.pfx) from disk to be used for server authentication.
```

File: docs/user-guide/station-credential-management.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+Two environment variables need to be set for making this happen:
+
+- **LNS_SERVER_PFX_PATH**: It's the absolute path to the .pfx certificate in the IoT Edge module filesystem (i.e.: '/var/lorastarterkit/certs/lns.pfx')
+- **LNS_SERVER_PFX_PASSWORD** *(optional)*: needs to be set if the .pfx was exported with password
+
+As stated above, LNS_SERVER_PFX_PATH is a path inside the IoT Edge module filesystem.
```
Comment (p-schuler) [inline docs/user-guide/station-credential-management.md:18]:
```suggestion
```

redundant?

File: docs/user-guide/station-credential-management.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+3. Set the 'LoRaWanNetworkSrvModule' HostConfiguration to include a binding for the folder
+
+    ```json
+    "Binds":  [
+    "/mnt/lora/certs/:/var/lorastarterkit/certs/"
+    ]
+    ```
+
+Additional reference on this process can be found [here](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-access-host-storage-from-module?view=iotedge-2020-11).
```
Comment (p-schuler) [inline docs/user-guide/station-credential-management.md:34]:
```suggestion
Additional information on this process can be found in the documentation - [Use IoT Edge device local storage from a module](https://docs.microsoft.com/azure/iot-edge/how-to-access-host-storage-from-module?view=iotedge-2020-11).
```
make it a habit to match the title of the target document when referencing links. Also if you reference links from the docs, I recommend removing the local specific part.

File: docs/user-guide/station-credential-management.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+### Importing 'tc.trust' in bundled 'LoRaBasicsStationModule'
+
+If you are making use of the bundled 'LoRaBasicsStationModule', it's possible to import a tc.trust certificate in the module itself.
+
+As in previous section, assuming the 'tc.trust' certificate (PEM) is located in a folder on the host OS at /mnt/lora/certs, you can 'bind' this path to one in the IoT Edge module itself. In order to do so:
+
+1. Log into your Azure Portal
```
Comment (p-schuler) [inline docs/user-guide/station-credential-management.md:42]: seems to be repeated from above?

**Review by p-schuler — APPROVED** [review]
I would remove the duplicate section about the binding for the 2 different certificates. The binding is described well in the first section and also linked to the official documentation.

**Pull request merged**